Sources/MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -692,10 +684,21 @@ open class CarPlayNavigationViewController: UIViewController, BuildingHighlighti
         let nextStep = progress.currentLegProgress.stepIndex + 1
         
         navigationMapView?.addArrow(route: progress.route, legIndex: legIndex, stepIndex: nextStep)
-        navigationMapView?.show([progress.route], legIndex: legIndex)
+        updateRouteLine(routeProgress: progress, coordinate: navigationService.router.location?.coordinate)
```

Comment:
Avoid the usage of `navigationMapView.show(::legIndex:)` during active navigation in `mobile` and `CarPlay`, because we may have the `routeLineTracksTraversal` turned on. Instead, packing the route line update to `updateRouteLine(routeProgress:coordinate:)` to avoid this issue.